src/calculator/ivs.ts

```diff
@@ -7,24 +7,24 @@ import { getCpm, getCp, getCpFromLevel } from './stats'
 export const getDefaultIvs = (
   dexEntry: PokedexEntry, cpLimit: number
 ): LevelStats => {
-  let stats: LevelStats = { level: 40, atk: 15, def: 15, hp: 15 }
+  let stats: LevelStats = { level: 40, atk: 15, def: 15, hp: 15 };
 
-  const maxCp = getCpFromLevel( dexEntry.baseStats, stats, 40 )
+  const maxCp = getCpFromLevel( dexEntry.baseStats, stats, 40 );
 
-  if ( maxCp <= cpLimit ) return stats
+  if ( maxCp <= cpLimit ) return stats;
 
   const combinations = getIvCombinations(
     dexEntry, 'overall', 1, 4096, cpLimit
-  )
+  );
 
-  const defaultIndex = Math.floor( combinations.length * .12207 )
+  const defaultIndex = Math.floor( combinations.length * 0.12207 );
```

Comment:
Fine with this but we should make sure we're consistent - I do prefer the leading 0